aspnetcore/fundamentals/error-handling.md

```diff
@@ -45,6 +45,9 @@ To configure a custom error handling page for the [Production environment](xref:
 * Catches and logs unhandled exceptions.
 * Re-executes the request in an alternate pipeline using the path indicated. The request isn't re-executed if the response has started. The template generated code re-executes the request using the `/Error` path.
 
+> [!WARNING]
+> An unhandled exception in the alternate pipeline is **not** propagated back to the original pipeline. The exception caught by <xref:Microsoft.AspNetCore.Builder.ExceptionHandlerExtensions.UseExceptionHandler%2A> is propagated instead.
```

Comment:
```suggestion
> An unhandled exception in the alternate pipeline is **not** propagated back to the original pipeline. The original exception caught by <xref:Microsoft.AspNetCore.Builder.ExceptionHandlerExtensions.UseExceptionHandler%2A> is propagated instead.
```

I'm not 100% on this, because "original" is used right before this re the pipeline, but I think it might be good to call out that it's the original exception being thrown.

Reply:
I find this saying the same thing twice. I think we can drop the 2nd sentence. Also, it isn't "back to" the original pipeline. It's "propagated further in" the original pipeline.